src/internal/serde/interface.go

```diff
@@ -138,40 +138,3 @@ type Encoder interface {
 	// TODO(msteffen) same comment re: proto.Message as for EncodeProto()
 	EncodeProtoTransform(proto.Message, func(map[string]interface{}) error) error
 }
-
```

Comment:
This interface unified JSON and YAML decoding for PipelineManifestReader, but since we're switching to YAML parsing only, there's only one implementation of the interface and it's no longer really necessary.